packages/graphql/mixin.server.js

```diff
@@ -20,6 +20,18 @@ const fetch = require('cross-fetch');
 
 let introspectionResult = undefined;
 
+const getApolloError = error => {
```

Comment:
I think the name is a little bit misleading as we don't get (return) an Apollo error here. We convert or wrap it...